components/atom/table/src/index.js

```diff
@@ -7,24 +7,39 @@ const CELL_TYPE = {
   data: 'td'
 }
 
-const AtomTable = ({head, body, foot, fullWidth}) => {
+const CELL_PADDING = {
+  xs: 'sizeXS',
```

Comment:
This naming convention `sizeXS` seems to be new, has there been an agreement about it? Perhaps it's enough to call it just `xs`.